client/src/client.rs

```diff
@@ -1265,6 +1266,25 @@ impl<B, E, Block, RA> Client<B, E, Block, RA> where
 		self.backend.blockchain().justification(*id)
 	}
 
+	/// Set offchain storage under given key and prefix.
+	pub fn set_offchain_storage(&self, prefix: &[u8], key: &[u8], value: &[u8]) -> sp_blockchain::Result<()> {
```

Comment:
I'd prefer to avoid going through the `Client` and use `OffchainWorkers` or `OffchainStorage` directly in the RPC.

Reply:
Ok, I'll try to use OffchainWorkers.